src/deepqmc/wf/baseline/pyscfext.py

```diff
@@ -27,6 +32,12 @@ def pyscf_from_mol(mol, basis, cas=None, **kwargs):
         spin=mol.spin,
         cart=True,
         parse_arg=False,
+        ecp=dict(
```

Comment:
I'd suggest this refactor:
```suggestion
        ecp={int(charge): mol.pp_type for charge in mol.charges[mol.pp_mask]}
```

Reply:
This shorter code looks much better. Changed in 98009b312db1bc5afdac5e8bebe35794f77e2827